app/assets/javascripts/oxalis/store.js

```diff
@@ -381,6 +381,7 @@ export const defaultState: OxalisState = {
     description: null,
     displayName: "Awesome Test Dataset",
     allowedTeams: [],
+    isDummyDataset: true,
```

Comment:
The solution is not optimal, but once we have a `logo` attribute we can set it for the dummy dataset to null to avoid flickering the default logo.